lib/osf-components/addon/components/file-embed-menu/component.ts

```diff
@@ -0,0 +1,72 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { htmlSafe } from '@ember/template';
+import Component from '@glimmer/component';
+import config from 'ember-get-config';
+import Intl from 'ember-intl/services/intl';
+import Toast from 'ember-toastr/services/toast';
+
+import FileModel from 'ember-osf-web/models/file';
+
+interface Args {
+    file: FileModel;
+}
+
+export default class FileEmbedMenuComponent extends Component<Args>{
+
+    @service toast!: Toast;
+    @service intl!: Intl;
+
+    get shareiFrameDynamic() {
+        const mfrStaticUrl = config.OSF.renderUrl.replace('/render', '/static');
+
+        return htmlSafe(`<style>
+    .embed-responsive {
+        position:relative;
```

Comment:
Hi Brian, will returning CSS result in external style sheets' formatting being overwritten?